### PR TITLE
fix #10

### LINKED ISSUE #10: CI does not fail when command fails
See my tests running here:
https://github.com/Malacath-92/shader-toy/actions/runs/117158083

You can see that the tests are failing (i.e. exiting with code 1) but the CI does not fail. I assume this is an issue within xvfb-action not forwarding the return code properly.
When downgrading to xvfb-action@v1.0 CI fails as expected
https://github.com/Malacath-92/shader-toy/actions/runs/117174757

### PR DESCRIPTION
**Pull request by Akuli**
fixes #10

### CODE REVIEW

**Comment by felipecrs** [comment]
I wonder how this fixes the issue... (I did not understand the code)

File: index.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ async function cleanUpXvfb() {
 }
 
 async function runCommand(command, directory) {
```
Comment (felipecrs) [inline index.js:40]: How about:

```suggestion
async function runCommand(command, directory) => (directory ? exec.exec(command, directory) : exec.exec(command))
```

**Comment by felipecrs** [comment]
Can you merge this @GabrielBB?

**Comment by GabrielBB** [comment]
@felipecrs Wow, I never saw a notification from Github for this project. I would've merged this immediately.

@Akuli Thanks!! Published -> https://github.com/marketplace/actions/gabrielbb-xvfb-action

**Comment by felipecrs** [comment]
Awesome! Tested and it worked. Thank you!

**Comment by GabrielBB** [comment]
@felipecrs Please upgrade to [v1.4](https://github.com/marketplace/actions/gabrielbb-xvfb-action). I just merged another fix -> https://github.com/GabrielBB/xvfb-action/pull/9

**Comment by felipecrs** [comment]
> @felipecrs Please upgrade to [v1.4](https://github.com/marketplace/actions/gabrielbb-xvfb-action). I just merged another fix -> #9

For sure! If you don't mind, please take a look at https://github.com/GabrielBB/xvfb-action/issues/13.

**Pull request merged**